From: Achill Gilgenast <achill@achill.org>
Date: Tue, 17 Mar 2026 21:53:52 +0100
Subject: data: Remove gsd-wacom as required component from session

The Wacom g-s-d plugin has been removed and therefore Phosh would fail to
launch since the dependency cannot be fullfilled.

Ref: https://gitlab.gnome.org/GNOME/gnome-settings-daemon/-/merge_requests/416
Signed-off-by: Achill Gilgenast <achill@achill.org>
Part-of: <https://gitlab.gnome.org/World/Phosh/phosh/-/merge_requests/1888>
(cherry picked from commit b0dae86755d3fcb947e10efec02e8d513498134e)
---
 data/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index da2e4fa..4341828 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -13,7 +13,6 @@ gsd_required_components = [
   'org.gnome.SettingsDaemon.Smartcard',
   'org.gnome.SettingsDaemon.Sound',
   'org.gnome.SettingsDaemon.UsbProtection',
-  'org.gnome.SettingsDaemon.Wacom',
   'org.gnome.SettingsDaemon.Wwan',
 ]
 
